src/importkeydialog.cpp

```
@@ -0,0 +1,141 @@
+
+  if (fingerprint.isEmpty()) {
+    showError(tr("Could not parse imported key fingerprint from GPG output."));
+    return false;
+  }
+
+  m_importedFingerprint = fingerprint;
+  showSuccess(fingerprint);
+  return true;
+}
+
+QString ImportKeyDialog::parseGpgImportOutput(const QString &output) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)